examples/stencil/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "build": "stencil build",
+    "watch": "stencil build --dev --watch --serve"
+  },
+  "dependencies": {
+    "vegemite": "file:../../"
+  },
+  "devDependencies": {
+    "@stencil/core": "^1.12.2",
+    "workbox-build": "4.3.1"
```

Comment:
The CLI auto-installed `workbox-build` whenever I ran the `build` script. Figure might as well include it :)

Reply:
Personally, i just let the CLI handle deps itself, but I don't think this hurts

Reply:
Right, the CLI installed this automatically & added it to the `package.json` file

Reply:
oh, yeah definitely fine to add it then :)